pootle/apps/pootle_app/views/admin/dashboard.py

```diff
@@ -339,13 +113,20 @@ def rq_stats():
     return result
 
 
+def checks():
+    from django.core.checks.registry import registry
+
+    ret = []
```

Comment:
@julen Is not enough with `return sum([check() for check in registry.registered_checks])`?